### DISCUSSION TITLE
Github should communicate more about Mobile View

### DISCUSSION THREAD

**Discussion by gpotter2** [community/community#21817] [New to GitHub]
<p>Hi.<br>
I couldn’t find any other mention of Github’s mobile view elsewhere than in <a href="https://github.blog/2013-07-15-github-s-on-your-phone/" rel="nofollow noopener">https://github.blog/2013-07-15-github-s-on-your-phone/</a><br>
It would be great if we could get some updates on what’s happening there. Let me explain</p>
<p>If you have been using this feature for the past year, you would have noticed that Github is gradually replacing those pages with fully responsive ones, used with both PCs and Mobile.</p>
<p>For instance, the file viewer was updated to be the same as the desktop one some time last year. The “issue” pages were also updated last year, but not the “PR” pages. The issue and PR listing was apparently updated yesterday.</p>
<p>Those features are rolled out without a mention on any blog, and while I totally understand why it’s simpler to maintain a single shared code base, it would be great to stay informed. There are issues with this migration.</p>
<p>For instance, on mobile using the old templates (still available in the PR pages), you can see who upvoted you’re post. But it you use the NEW template in an issue page, then you can’t ! It also renders super weirdly on the newer templates… it’s barely usable</p>
<p>Old: </p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/13d9eb8d153e1719d736228323728b92fc957dda.jpeg" data-download-href="/uploads/short-url/2PBXKfIOuSqlMgsW360t7sP4r6G.jpg?dl=1" title="26647B31-09F5-44AA-95FF-00990CD19243.jpeg" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/10530980/181066999-79caf5a6-44e2-498b-a52e-b91f1943f50f.jpg" alt="26647B31-09F5-44AA-95FF-00990CD19243.jpeg" data-base62-sha1="2PBXKfIOuSqlMgsW360t7sP4r6G" width="690" height="367" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/13d9eb8d153e1719d736228323728b92fc957dda_2_690x367.jpg, https://aws1.discourse-cdn.com/github/original/1X/13d9eb8d153e1719d736228323728b92fc957dda.jpeg 1.5x, https://aws1.discourse-cdn.com/github/original/1X/13d9eb8d153e1719d736228323728b92fc957dda.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/13d9eb8d153e1719d736228323728b92fc957dda_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">26647B31-09F5-44AA-95FF-00990CD19243.jpeg</span><span class="informations">828×441 105 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/61a809ff3e4155073857af3388c49de4372d95b4.jpeg" data-download-href="/uploads/short-url/dVUkS2H4FGwjn3rzLMm6JRJIBRW.jpg?dl=1" title="749C24F1-849C-476C-89A4-4E10CC82CCF9.jpeg" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/10530980/181067001-7f42e492-99de-4c01-88e0-7b0b669ba741.jpg" alt="749C24F1-849C-476C-89A4-4E10CC82CCF9.jpeg" data-base62-sha1="dVUkS2H4FGwjn3rzLMm6JRJIBRW" width="690" height="412" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/61a809ff3e4155073857af3388c49de4372d95b4_2_690x412.jpg, https://aws1.discourse-cdn.com/github/original/1X/61a809ff3e4155073857af3388c49de4372d95b4.jpeg 1.5x, https://aws1.discourse-cdn.com/github/original/1X/61a809ff3e4155073857af3388c49de4372d95b4.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/61a809ff3e4155073857af3388c49de4372d95b4_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">749C24F1-849C-476C-89A4-4E10CC82CCF9.jpeg</span><span class="informations">828×495 108 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>New: </p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/7cfa2e3d5a3c4df119525dca0fe075da783a380c.jpeg" data-download-href="/uploads/short-url/hPBdnJ3GfBNpaflqc9QhmPIvENC.jpg?dl=1" title="DC311607-1AFF-49FA-8ED0-302339C5BD2F.jpeg" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/10530980/181067002-899d10ba-3a7a-4aec-8f67-ada6a911938d.jpg" alt="DC311607-1AFF-49FA-8ED0-302339C5BD2F.jpeg" data-base62-sha1="hPBdnJ3GfBNpaflqc9QhmPIvENC" width="690" height="452" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/7cfa2e3d5a3c4df119525dca0fe075da783a380c_2_690x452.jpg, https://aws1.discourse-cdn.com/github/original/1X/7cfa2e3d5a3c4df119525dca0fe075da783a380c.jpeg 1.5x, https://aws1.discourse-cdn.com/github/original/1X/7cfa2e3d5a3c4df119525dca0fe075da783a380c.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/7cfa2e3d5a3c4df119525dca0fe075da783a380c_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">DC311607-1AFF-49FA-8ED0-302339C5BD2F.jpeg</span><span class="informations">828×543 84.7 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/a65b67b019f3e841daa07cea346396ebc760f8ea.jpeg" data-download-href="/uploads/short-url/nJF5R8PGcqQ4l7h3iST943pw9p8.jpg?dl=1" title="FA2C6404-32C3-4734-A66E-59DD4C7E88F2.jpeg" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/10530980/181067003-cc25a6cf-a3d0-476b-904b-4b986617eaad.jpg" alt="FA2C6404-32C3-4734-A66E-59DD4C7E88F2.jpeg" data-base62-sha1="nJF5R8PGcqQ4l7h3iST943pw9p8" width="690" height="267" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/a65b67b019f3e841daa07cea346396ebc760f8ea_2_690x267.jpg, https://aws1.discourse-cdn.com/github/original/1X/a65b67b019f3e841daa07cea346396ebc760f8ea.jpeg 1.5x, https://aws1.discourse-cdn.com/github/original/1X/a65b67b019f3e841daa07cea346396ebc760f8ea.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/a65b67b019f3e841daa07cea346396ebc760f8ea_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">FA2C6404-32C3-4734-A66E-59DD4C7E88F2.jpeg</span><span class="informations">828×321 86.9 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Yesterday’s update is also still looks experimental: there is a weird white line displayed at the top. </p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/d2f74065b3ad53c3013bd936178345e2a9cb15ac.jpeg" data-download-href="/uploads/short-url/u6i0aLyUGRaYaeNEOtZAOeOPj8E.jpg?dl=1" title="723E4DF5-007B-49DB-9476-8B554283A9C0.jpeg" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/10530980/181067004-ee39173f-91b2-4e11-830f-8d2efbef6a61.jpg" alt="723E4DF5-007B-49DB-9476-8B554283A9C0.jpeg" data-base62-sha1="u6i0aLyUGRaYaeNEOtZAOeOPj8E" width="690" height="334" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/d2f74065b3ad53c3013bd936178345e2a9cb15ac_2_690x334.jpg, https://aws1.discourse-cdn.com/github/original/1X/d2f74065b3ad53c3013bd936178345e2a9cb15ac.jpeg 1.5x, https://aws1.discourse-cdn.com/github/original/1X/d2f74065b3ad53c3013bd936178345e2a9cb15ac.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/d2f74065b3ad53c3013bd936178345e2a9cb15ac_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">723E4DF5-007B-49DB-9476-8B554283A9C0.jpeg</span><span class="informations">828×401 65 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Could we get news about those changes? Maybe an option to stick to the old theme until the new one works properly ? (a “beta” feature of some sort) That would really be awesome.</p>
<p>Thanks</p>

**Comment by logankilpatrick** [comment, marked as answer]
<p>Hey! I suggest following the specific folks making these changes on Twitter. I saw the new changes the rolled out since I follow a bunch of the engineers from GitHub on Twitter. My guess is that since these are incremental improvements in most cases, they are not bothering to do blog posts though I do think the devs are quite responsive on Twitter if you have bugs to report. </p>

**Comment by gpotter2** [comment]
<p>That’s great advice.<br>
If you happen to know who to follow that’d be great, but I’ll probably find them eventually.<br>
Thanks</p>

**Comment by Levminer** [comment]
<p>Use the <a href="http://github.com/mobile" rel="nofollow noopener">mobile app.</a> Its so much better then the web version.</p>
